src/eva/plotting/emcpy/plot_tools/figure_driver.py

```diff
@@ -65,27 +65,40 @@ def figure_driver(config, data_collections, timing, logger):
         if batch_conf:
             # Get potential variables
             variables = batch_conf.get('variables', [])
-            # Get list of channels
+
+            # Get list of channels and load step variables
             channels_str_or_list = batch_conf.get('channels', [])
             channels = parse_channel_list(channels_str_or_list, logger)
 
+            step_vars = channels if channels else ['none']
+            step_var_name = 'channel'
+            title_fill = ' Ch. '
+
+            # Get list of levels, conditionally override step variables
+            levels_str_or_list = batch_conf.get('levels', [])
+            levels = parse_channel_list(levels_str_or_list, logger)
+            if levels:
+                step_vars = levels
+                step_var_name = 'level'
+                title_fill = ' Lv. '
```

Comment:
personal nit pick, would prefer `' Lev. '` , I think the one extra letter makes it less ambiguous, but curious what others think.

Reply:
Done.